neo/renderer/RenderSystem_init.cpp

```diff
@@ -278,7 +278,12 @@ PFNGLPROGRAMLOCALPARAMETER4FVARBPROC	qglProgramLocalParameter4fvARB;
 PFNGLDEPTHBOUNDSEXTPROC                 qglDepthBoundsEXT;
 
 // DG: couldn't find any extension for this, it's supported in GL2.0 and newer, incl OpenGL ES2.0
+// SE: only the ATI implementation is provided on Mac OS X 10.4 and 10.5
+#if OSX_TIGER || OSX_LEOPARD
```

Comment:
`#if OSX_TIGER || OSX_LEOPARD` most probably produces compiler warnings when they are not `#defined` (which they certainly won't be on non-apple platforms).

Do you really want to check their value (if OSX_TIGER is 0 or 1/non-zero, this is what `#if` does), or is checking for their existence enough (`#if defined(OSX_TIGER) || defined(OSX_LEOPARD)`)?